x/chainmain/client/cli/genaccounts.go

```diff
@@ -110,8 +111,10 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 
 			balances := banktypes.Balance{Address: addr.String(), Coins: coins.Sort()}
 			baseAccount := authtypes.NewBaseAccount(addr, nil, 0, 0)
+			var vestingAddress string
 
 			if !vestingAmt.IsZero() {
```

Comment:
vestingAmount?

Reply:
You mean rename `vestingAddress` to `vestingAmount`?

Reply:
renaming vestingAmt -> vestingAmount

Reply:
Yeah. Maybe we can do that in a separate PR as this one does not change that part of the code.